waspc/cli/Main.hs

```diff
@@ -44,6 +46,7 @@ main = do
     Command.Call.Version -> printVersion
     Command.Call.Build -> runCommand build
     Command.Call.Telemetry -> runCommand Telemetry.telemetry
+    Command.Call.Deps -> runCommand deps
     Command.Call.Unknown _ -> printUsage
 
   -- If sending of telemetry data is still not done 1 second since commmand finished, abort it.
```

Comment:
You should add `deps` under `printUsage`, so that people can learn about it.

Reply:
Added!